### PR TITLE
Bump language and mod enums

### PR DESCRIPTION
**Pull request by iotanum**
- Add ScoreV2 mod enum
- Add various new language enums

### CODE REVIEW

**Comment by iotanum** [comment]
@khazhyk

**Comment by khazhyk** [comment]
What are you basing this on? This doesn't seem to match https://github.com/ppy/osu-api/wiki

**Comment by iotanum** [comment]
Didn't seem to find this documented anywhere at osu! side so based via here:
For example: https://osu.ppy.sh/beatmapsets?l=2
l=2 argument, 2 according to wiki is english lang, 5 is instrumental and so on.

**Comment by iotanum** [comment]
Also added new genres according to this:
image
Click on whatever genre/language in this GUI to see it's API num via search query

File: osuapi/enums.py [hunk 1]
```diff
@@ -147,12 +149,16 @@ class BeatmapGenre(Enum):
...
+    classical = 12
+    folk = 13
+    jazz = 14
 
 
 class BeatmapLanguage(Enum):
     """Enum represeting the language of a beatmap."""
     any = 0
-    other = 1
+    unspecified = 1
```
Comment (khazhyk) [inline osuapi/enums.py:161]: So was the old api inaccurate? have you tried fetching sample beatmap sets and checking that the api number matches? If this is correct, cool, it's just strange it contradicts the wiki

Comment (iotanum) [reply osuapi/enums.py:161]: Yeah, no problem.
[This](https://osu.ppy.sh/api/get_beatmaps?b=2011121) [(beatmap)](https://osu.ppy.sh/beatmapsets/960601#osu/2011121) returns `"language_id":"1"`, unspecified.
[This](https://osu.ppy.sh/api/get_beatmaps?b=1991102) [(beatmap)](https://osu.ppy.sh/beatmapsets/945922#osu/1991102) returns `"language_id":"14"`, other.
[This](https://osu.ppy.sh/api/get_beatmaps?b=1906448) [(beatmap)](https://osu.ppy.sh/beatmapsets/912629#osu/1906448) returns `"language_id":"13"`, polish.

My guess is that when they introduced the 'unspecified' category they moved 'other' to another number... maybe

**Pull request merged**